source/code/plugins/patch_management_lib.rb

```diff
@@ -180,6 +180,19 @@ def availableUpdatesXMLtoHash(availableUpdatesXML, os_short_name)
         ret
     end
 
+    def availableHeartBeatData()        
+        ret = {}
+        ret["CollectionName"] = "HeartBeatData" + @@delimiter + 
+                                "0.0.UpdateManagement.0"+ @@delimiter + "HeartBeat"
+        ret["PackageName"] = "UpdateManagementHeartBeat"
+        ret["Architecture"] = nil
```

Comment:
I just realized, if we do not set Architecture, complex key in ingestion will not work. Let's use value "all"